### ISSUE TITLE
Add information about accessibility? 

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/TYPO3CMS-Tutorial-Editors#67]
> The backend in v11 is optimized for accessibility in many places, but the knowledge about this has to be passed on via training. A separate documentation section "Accessibility" in the TYPO3 Guide for Editors should be created as a supplement.

https://typo3.org/article/typo3-and-its-accessibility-in-the-backend

Related: #28, keybord commands: https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/HelpInside/KeyboardCommands/Index.html

**Comment by linawolf** [comment]
https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/pull/95 Adds information about making the frontend accessible. the keyboard commands are also displayed.
